content/en/docs/concepts/security/pod-security-standards.md

```diff
@@ -209,11 +209,11 @@ well as lower-trust users.The following listed controls should be enforced/disal
 		<tr>
 			<td>Privilege Escalation</td>
 			<td>
-				Privilege escalation to root should not be allowed.<br>
+				Privilege escalation (such as via set-user-ID or set-group-ID file mode) should not be allowed.<br>
 				<br><b>Restricted Fields:</b><br>
-				spec.containers[*].securityContext.privileged<br>
-				spec.initContainers[*].securityContext.privileged<br>
-				<br><b>Allowed Values:</b> false, undefined/nil<br>
+				spec.containers[*].securityContext.allowPrivilegeEscalation<br>
+				spec.initContainers[*].securityContext.allowPrivilegeEscalation<br>
+				<br><b>Allowed Values:</b> false<br>
```

Comment:
Looking at the SecurityContext resource, should the "allowed values" include undefined/nil?
Minor pt, but wondering why this has been excluded.
I also referre to this community doc:  
https://github.com/kubernetes/community/blob/master/contributors/design-proposals/auth/no-new-privs.md#pod-security-policy-changes

Reply:
I removed undefined/nil because I believe "false only" is the setting most in the spirit of the 'restricted' policy, especially the phrases "at the expense of some compatibility" and "Privilege escalation to root should not be allowed" in the current version of the doc.

This is related to the comment in the design proposal, "when a container is not run as root, uid 0 (to prevent breaking suid binaries)". In the "restricted" policy, we specifically want to break set-user-ID binaries to prevent them from being used as a bypass for runAsNonRoot et al.

Thanks for asking!